src/in_json.c

```diff
@@ -2192,7 +2192,8 @@ _set_struct_field (Bit_Chain *restrict dat, const Dwg_Object *restrict obj,
                   const size_t blen = len / 2;
                   // binary hexstring
                   BITCODE_TF buf = (BITCODE_TF)malloc (blen);
-                  if ((len = in_hex2bin (buf, str, blen) != blen))
+                  len = in_hex2bin (buf, str, blen);
+                  if ((len != blen))
```

Comment:
No (( please

Reply:
In case of ```len = in_hex2bin (buf, str, blen) != blen``` len is 0 or 1 depending on result of ```in_hex2bin (buf, str, blen) != blen```.
I don't think that we need it.

if we have:
```
const size_t k = dwg->header.from_version > R_2004 ? 512 : 256;
```
and
```
if (len > k)
...
else if (len != k)
```

Or am I missing something?

Reply:
Just the double parens

Reply:
uff